services/parser-database/connector.py

```diff
@@ -20,20 +19,24 @@ def get_documents_to_parse():
 
 
 def get_keywords(text):
-    """Get keywords that relate to this article
+    """Get keywords that relate to this article (from NLP service)
 
     Args:
         text (sting): text to extract keywords from
 
     Returns:
         [list]: list of extracted keywords
     """
-    splited_text = text.split()
-    keywords = [
-        splited_text[random.randint(0, len(splited_text) - 1)],
-        splited_text[random.randint(0, len(splited_text) - 1)],
-    ]
-    return keywords
+    extracted_keywords = []
+    request = {'text': text}
+    nlp_output = requests.post(os.getenv("KEYWORDS_SERVICE"), json=request)
+    json_output = nlp_output.json()
+    if 'error' in json_output:
+        raise Exception(json_output['error']['message'])
+    for keyword in json_output["tokens"]:
+        extracted_keywords.append(keyword["lemma"])
+        print(extracted_keywords)
```

Comment:
logging?

Reply:
+1

That doesn't display anything on the container logs right?

Reply:
Oops